spec/models/book_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  describe "validations" do
+    let(:harry) { build(:book) }
+    context "complete book details" do
+      it "is valid" do
+        expect(harry).to be_valid
+      end
+    end
+
+    context "title not present" do
+      it "is invalid" do
+        harry.title = nil
+        harry.valid?
+        expect(harry.errors["title"]).to include("can't be blank")
+      end
+    end
+  end
+
+  describe "#search_by_term scope" do
+    before do
+      10.times do
```

Comment:
Do you really need so many objects to run this test?
The less object the better - not a problem now but when your app grows... Be scrappy fro the beginning :)

Reply:
Haha ok yeah you are right. I'll tone it down to just 2 books